bin/check-rabbitmq-queue-drain-time.rb

```diff
@@ -48,9 +49,9 @@ class CheckRabbitMQQueueDrainTime < Sensu::Plugin::Check::CLI
          short: '-v',
          long: '--vhost VHOST'
 
-  option :user,
+  option :username,
```

Comment:
I like making it consistent but this makes it a breaking change. This will need to be called out in the CHANGELOG.